package.json

```diff
@@ -2,7 +2,7 @@
   "name": "openapi-workspaces",
   "license": "MIT",
   "private": true,
-  "version": "0.47.14",
+  "version": "0.47.15-1",
```

Comment:
👀

Reply:
not planning on releasing a new release version yet - will keep it at the prerelease for any future prerelease bumps